docs/reproducing.md

```diff
@@ -28,6 +28,15 @@ If you are not sure how to build the fuzzer using the project's build system,
 you may also use Docker ([how?](installing_docker.md), [why?](faq.md#why-do-you-use-docker)) commands 
 to replicate the exact build steps used by OSS-Fuzz and then feed the reproducer input to the fuzz target.
 
+- *Pull the latest Docker images:*
+
+```bash
+$ python infra/helper.py pull_images
+```
+
+  Docker images get regularly updated. In some cases, a particular issue can be
```

Comment:
updated with latest version of clang compiler binaries ?

Reply:
not necessary clang binaries, I think, e.g. in case of build checks it was our script used in base-runner image. IIUC, we rebuild and update base images once a day, even if nothing has changed for them.

Reply:
yes then then add as example

updated with X, Y, etc to give some useful context on why it is done.

Reply:
Also pull images can be moved into its own section at start, just to do it for everything, upto you.

Reply:
Done and Done!